plugins/psv-extractor/package.json

```diff
@@ -9,14 +9,15 @@
     "node": ">= 12"
   },
   "source": "src/index.ts",
-  "main": "dist/main.js",
+  "main": "dist/index.js",
```

Comment:
We should standardize our plugins bundling names - some use `index.js`, `main.js`, `main.mjs`, etc - so we don't trip on this again.